client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -20,18 +22,21 @@ class NameserversToggle extends PureComponent {
 	render() {
 		return (
 			<Card compact className="name-servers__dns">
-				<span className="name-servers__title">
-					{ this.props.translate( 'Use WordPress.com Name Servers' ) }
+				<span className="name-servers__title" onClick={ this.handleToggle }>
+					{ NAMESERVERS_TOGGLE_TITLE }
```

Comment:
What if we remove this `span` and add the label text to the toggle component? Then style the label to look like this span looked. So essentially, using a sole `ToggleControl` for the purpose. Have you tried that? I believe it should solve all the problems we are facing.

Reply:
You are right. I have just done this along with some styling,  please let me know.

Reply:
Thanks, @spraveenitpro, I'm going to take a look at the updated code on Monday.